frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tabs/details.spec.ts

```diff
@@ -31,72 +32,99 @@ describe('Test VM details tab', () => {
-      cy.get(detailsTab.vmStatus).should('contain', VM_STATUS.Running);
-      cy.get(detailsTab.vmPod).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
-      cy.get(detailsTab.vmIP).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
-      cy.get(detailsTab.vmHostname).should('contain', YAML_VM_NAME);
-      cy.get(detailsTab.vmTimezone).should('contain', 'EDT');
-      cy.get(detailsTab.vmNode).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
-      cy.get(detailsTab.activeUser).should('contain', 'No Active Users');
-    }
+      cy.get(detailsTab.vmStatus).should('contain', VM_STATUS.Stopped);
+      cy.get(detailsTab.vmPod).should('contain', DEFAULT_VALUES.NOT_AVAILABLE);
+      cy.get(detailsTab.vmIP).should('contain', DEFAULT_VALUES.NOT_AVAILABLE);
+      cy.get(detailsTab.vmHostname).should('contain', DEFAULT_VALUES.VM_NOT_RUNNING);
+      cy.get(detailsTab.vmTimezone).should('contain', DEFAULT_VALUES.VM_NOT_RUNNING);
+      cy.get(detailsTab.vmNode).should('contain', DEFAULT_VALUES.NOT_AVAILABLE);
+      cy.get(detailsTab.activeUser).should('contain', DEFAULT_VALUES.VM_NOT_RUNNING);
+    });
+
+    it('ID(CNV-4037) Check VM details while VM is running', () => {
+      // make it downstream only as VM created from yaml have guest agent enabled.
+      if (Cypress.env('DOWNSTREAM')) {
+        // wait for guest agent ready before move to details tab
+        tab.navigateToOverview();
+        detailViewAction(VM_ACTION.Start);
+
+        // Status card
+        cy.contains(dashboardTab.vmHealth, VM_STATUS.Starting, {
+          timeout: VM_ACTION_TIMEOUT.VM_IMPORT_AND_BOOTUP,
+        }).should('exist');
+        cy.contains(dashboardTab.vmHealth, VM_STATUS.Running, {
+          timeout: VM_ACTION_TIMEOUT.VM_IMPORT_AND_BOOTUP,
+        }).should('exist');
+        cy.get(dashboardTab.guestAgentOK, { timeout: 180000 }).should('exist');
+
+        tab.navigateToDetails();
+        cy.get(detailsTab.vmName).should('contain', YAML_VM_NAME);
+        cy.get(detailsTab.vmNS).should('contain', testName);
+        cy.get(detailsTab.vmOS).should('contain', 'Red Hat Enterprise Linux');
+        cy.get(detailsTab.vmTemplate).should('contain', 'rhel8-server-tiny');
+        cy.get(detailsTab.vmStatus).should('contain', VM_STATUS.Running);
+        cy.get(detailsTab.vmPod).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
+        cy.get(detailsTab.vmIP).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
+        cy.get(detailsTab.vmHostname).should('contain', YAML_VM_NAME);
+        cy.get(detailsTab.vmTimezone).should('contain', 'EDT');
+        cy.get(detailsTab.vmNode).should('not.contain', DEFAULT_VALUES.NOT_AVAILABLE);
+        cy.get(detailsTab.activeUser).should('contain', 'No Active Users');
+      }
+    });
+
+    it('ID(CNV-5320) Displays guest agent data in Disks tab', () => {
+      // make it downstream only as VM created from yaml have guest agent enabled.
```

Comment:
nitpick: 'VM' => 'VMs'

Reply:
This one still needs to be corrected.

Reply:
This is a miss one, would you mind if I correct it in other PR?